docs/API/AttributesUsage.md

```diff
@@ -0,0 +1,227 @@
+# Attributes usage
+
+## Overview
+
+Support to attributes is a feature that allows chaincode make use of extended data in a transaction certificate. These attributes are certified by the Attributes Certificate Authority (ACA) so the chaincode can trust in the authenticity of the attributes' values.
```

Comment:
"Support to attributes" ... ? Not quite sure what you mean? Maybe just "Attributes" ?